src/ray/object_manager/object_manager.cc

```diff
@@ -18,9 +18,10 @@ ObjectManager::ObjectManager(asio::io_service &main_service,
       buffer_pool_(config_.store_socket_name, config_.object_chunk_size),
       rpc_work_(rpc_service_),
       gen_(std::chrono::high_resolution_clock::now().time_since_epoch().count()),
-      object_manager_server_("ObjectManager", config_.object_manager_port),
+      object_manager_server_("ObjectManager", config_.object_manager_port,
+                             config_.rpc_service_threads_number),
       object_manager_service_(rpc_service_, *this),
-      client_call_manager_(main_service) {
+      client_call_manager_(rpc_service_) {
```

Comment:
This means the object manager will be called from multiple threads right, was the object manager already thread-safe?

Reply:
I believe the some of the data transfer code is, but not for the whole object manager.

Reply:
Let me verify this part and let you know.